RecipeApp/.idea/.idea.RecipeApp/.idea/riderModule.iml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not familiar enough with Rider to know if these files should be checked in or not but you should look it up if you haven't.